index.js

```diff
@@ -344,6 +327,30 @@ function buildParameter(oldParameter) {
     name: oldParameter.name,
     required: !!oldParameter.required
   };
+
+  if (primitiveTypes.indexOf(oldParameter.type) === -1) {
+    parameter.schema = {$ref: '#/definitions/' + oldParameter.type};
+  } else if (oldParameter.paramType === 'body') {
+    parameter.schema = buildParamType(oldParameter);
+  }else {
```

Comment:
JSCS should've catch this. `}else {` doesn't have space after closing brace. Can you fix this?